script/redis-traffic-stats

```diff
@@ -26,8 +26,7 @@ BEGIN {
     }
 }
 
-use Net::Pcap;
-use Net::Pcap::Reassemble;
+use Net::Pcap qw(:functions);
```

Comment:
Could you change to `use Net::Pcap;`? Because I got following error.

```
Undefined subroutine &main::pcap_compile called at /home/hirose31/repos/redis-traffic-stats/script/redis-traffic-stats line 108.
```